### PR TITLE
Make auth header prefix configurable

### PR DESCRIPTION
**Pull request by qpfmtlcp**
Inspired by #19

### CODE REVIEW

**Comment by johnraz** [comment]
I'm very much in favor of this.

**Comment by belugame** [comment]
@qpfmtlcp thank you for your contribution. I'm happy to add this. Please add test coverage, as in a unit test that confirms one can successfully login with the right prefix and not with the wrong prefix.

File: tests/tests.py [hunk 1]
```diff
@@ -44,6 +44,9 @@ def get_basic_auth_header(username, password):
 user_serializer_knox = knox_settings.defaults.copy()
 user_serializer_knox["USER_SERIALIZER"] = UserSerializer
 
+auth_header_prefix_knox = knox_settings.defaults.copy()
+auth_header_prefix_knox["AUTH_HEADER_PREFIX"] = 'Baerer'
```
[the remark this answers is not in the document]
Comment (qpfmtlcp) [reply tests/tests.py]: oops. I really appreciate your review.

**Comment by belugame** [comment]
nice work :+1: 
we sadly still cannot release anything to pypi due to #107

**Pull request merged**